packages/noise/src/index.ts

```diff
@@ -0,0 +1,43 @@
+import { random } from 'remotion';
+import {
+  createNoise2D as _createNoise2D,
+  createNoise3D as _createNoise3D,
+  createNoise4D as _createNoise4D,
+} from 'simplex-noise';
+import type {
+  NoiseFunction2D,
+  NoiseFunction3D,
+  NoiseFunction4D,
+} from 'simplex-noise';
+
+type RemotionSeed = number | string | null;
```

Comment:
@JonnyBurger I think it worth to consider making the `RemotionSeed` type available to export from `remotion`

Reply:
Logged an issue: https://github.com/remotion-dev/remotion/issues/1402